.travis.yml

```diff
@@ -8,27 +8,29 @@ services:
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION="4.0.x-dev"
+    - COMPOSER_ROOT_VERSION="3.0.x-dev"
 
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL PHPUNIT_TEST=1 RECIPE_VERSION="^1 --prefer-lowest"
-    - php: 7.0
-      env: DB=MYSQL PHPUNIT_TEST=1 RECIPE_VERSION="^1"
+      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
     - php: 7.1
-      env: DB=PGSQL PHPUNIT_TEST=1 RECIPE_VERSION="^4 --prefer-lowest"
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_COVERAGE_TEST=1 PDO=1
     - php: 7.2
-      env: DB=PGSQL PHPUNIT_COVERAGE_TEST=1 RECIPE_VERSION="^4"
+      env: DB=PGSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1
     - php: 7.3
-      env: DB=PGSQL PHPCS_TEST=1 PHPUNIT_TEST=1 RECIPE_VERSION="4.x-dev"
+      env: DB=MYSQL RECIPE_VERSION=4.6.x-dev PHPUNIT_TEST=1 
+    - php: 7.4
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1
 
 before_script:
   - phpenv rehash
   - phpenv config-rm xdebug.ini
 
   - composer validate
-  - composer require --no-update silverstripe/userforms:5.x-dev silverstripe/recipe-cms:$RECIPE_VERSION
+  - composer require --no-update silverstripe/recipe-cms:$RECIPE_VERSION silverstripe/userforms:^5
+  # Fix for running phpunit 5 on php 7.4+
+  - composer require --no-update sminnee/phpunit-mock-objects:^3
```

Comment:
Would be better as a dev dependnecy

Reply:
You could put it into recipe-testing and use that as a dev dependency.. I'd be more on board with that